apps/system/js/audio_channel_controller.js

```diff
@@ -50,6 +76,43 @@
       case '_destroyed':
         this.publish('destroyed');
         break;
+
+      case 'mozChromeEvent':
+        var detail = evt.detail;
+        var name = detail.name;
+        switch (detail.type) {
+          case 'system-audiochannel-state-changed':
+            if (name === this.name) {
+              this._states.active = detail.isActive;
+              this.publish('statechanged');
+            }
+            break;
+
+          case 'system-audiochannel-mute-onsuccess':
```

Comment:
Add `system-audiochannel-mute` to instead of `system-audiochannel-play` and `system-audiochannel-play` for baku's comments in https://bugzilla.mozilla.org/show_bug.cgi?id=1157140#c24.